README.md

```diff
@@ -135,6 +135,24 @@ how the resources are generated.
 --update-strategy
 # Action to perform when a property file already exists: 'update', 'recreate', 'fail', 'ask' (per file)
 ```
+
+`dbt-coves generate properties` supports the following args:
+
+```console
```

Comment:
You should use `shell` https://github.com/github/linguist/blob/master/lib/linguist/languages.yml

Reply:
So all current 'console' command usages must be 'shell', right?